3_javascript/javascript1/week2/script.js

```diff
@@ -0,0 +1,85 @@
+//Flight Booking Fullname
+function fullName(firstName, surName) {
+  return firstName + " " + surName;
+}
+const fullName1 = fullName("Anjali", "Kumari");
+const fullName2 = fullName("Babul", "Raja");
+console.log(fullName1);
+console.log(fullName2);
+
+//Formal Name
+function getFullname(firstName, surName, useFormalName, isWoman) {
+  if (useFormalName) {
+    return "Lord" + " " + firstName + " " + surName;
+  } else if (isWoman) {
+    return "Miss" + " " + firstName + " " + surName;
+  } else {
+    return firstName + " " + surName;
+  }
+}
+const formalName = getFullname("Anjali", "Kumari", false, true);
+console.log(formalName);
+//note: other way to solve the promblem is = isWoman ? 'Lady' : 'Lord';
+
+//Weather Wear
+function todayWeatherTemp(temp) {
```

Comment:
giving a short name for the props is a good idea, but sometimes it is important to give a good and understandable name
This comment is not to tell that what you did is wrong, it is just to let you do it in a better way.